### PR TITLE
LPS-111700 sf

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:gradle-plugins

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java [hunk 1]
```diff
@@ -272,6 +312,100 @@ public void setTargetPlatformVersion(Object targetPlatformVersion) {
 		_targetPlatformVersion = targetPlatformVersion;
 	}
 
+	public void setWorkspaceProductKey(Object workspaceProductKey) {
+		_product = workspaceProductKey;
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java:316]: The `product` property is named differently in a few places.  Can we rename it to `productKey` everywhere?

```
public void setProductKey(Object productKey) {
	_productKey = productKey;
}
```

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java [hunk 2]
```diff
@@ -272,6 +312,100 @@ public void setTargetPlatformVersion(Object targetPlatformVersion) {
...
+	private ProductInfo _getProductInfo(String productKey) {
+		if (productKey == null) {
+			return null;
+		}
+
+		return _productInfos.computeIfAbsent(
+			productKey,
+			key -> {
+				try {
+					DownloadCommand download = new DownloadCommand();
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java:374]: Full variable name to improve code readaility `downloadCommand`

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java [hunk 3]
```diff
@@ -272,6 +312,100 @@ public void setTargetPlatformVersion(Object targetPlatformVersion) {
...
+
+					download.setCacheDir(_workspaceCacheDir);
+					download.setUrl(new URL(_PRODUCT_INFO_URL));
+					download.setToken(false);
+					download.setUserName(null);
+					download.setPassword(null);
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java:380]: Sort alphabetically

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java [hunk 4]
```diff
@@ -272,6 +312,100 @@ public void setTargetPlatformVersion(Object targetPlatformVersion) {
...
+
+					download.execute();
+
+					Path downloadPath = download.getDownloadPath();
+
+					try (JsonReader reader = new JsonReader(
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java:386]: Full variable name to improve code readaility `jsonReader`

**Comment by petershin** [comment]
Hi @gamerson, please take a look at my comments.  The property name was referenced with three different names and I wasn't sure which one to use.

**Comment by petershin** [comment]
See https://github.com/petershin/liferay-portal/pull/997

**Comment by gamerson** [comment]
> Hi @gamerson, please take a look at my comments. The property name was referenced with three different names and I wasn't sure which one to use.

Everything looks good, thanks!

**Pull request closed without merging**